typescript/pnpm-workspace.yaml

```diff
@@ -0,0 +1,6 @@
+packages:
+  - "agentkit"
+  - "create-onchain-agent"
+  - "create-onchain-agent/templates/*"
```

Comment:
I figured a minor devx improvement could be made here, so contributors do not need to remember to add to this list. However, I am open to reversing this back to individually listing each package if we want to be explicit

Reply:
(For reference, this list used to live in package.json)